images/nginx/rootfs/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -d "/modules_mount/etc/nginx" ]; then
+	for dir in /modules_mount/etc/nginx/*; do
+		cp "$dir"/* "/etc/nginx/$(basename "$dir")"
+	done
+fi
+if [ -d "/modules_mount/usr/local" ]; then
+	for dir in /modules_mount/usr/local/*; do
+		cp "$dir"/* "/usr/local/$(basename "$dir")"
+	done
+fi
```

Comment:
And same here :)

right now we probably just want to copy the modules (/etc/nginx/modules) and nothing else. We can drop this, and re-visit once we decide to move mod_security